assignments/codingStyle.md

```diff
@@ -23,4 +22,23 @@ else
 ## Solution
 ```js
 // your code goes here
-```
\ No newline at end of file
+```
+function pow(x, n) {
+    let result=1;
```

Comment:
Space between variable name, assignment operator and the value. Go through the image below again and refactor it again.

![](http://javascript.info/article/coding-style/code-style@2x.png)